_includes/api/en/5x/routing-args.html

```diff
@@ -26,18 +26,19 @@ <h4> Arguments</h4>
 <tr>
 <td> <code>callback</code></td>
 <td>
-Callback functions; can be:
+A series of one or more functions that are invoked with the following arguments: <code>request</code>, <code>response</code>, <code>next</code>. Each callback optionally sends a response to the client or passes control to the next callback in the series by calling `next()`. `callback` can be provided as:
 <ul>
-<li>A middleware function.</li>
-<li>A series of middleware functions (separated by commas).</li>
+<li>One or more middleware functions.</li>
 <li>An array of middleware functions.</li>
-<li>A combination of all of the above.</li>
+<li>Any combination of the above.</li>
 </ul>
 <p>
-You can provide multiple callback functions that behave just like middleware, except
-that these callbacks can invoke <code>next('route')</code> to bypass
-the remaining route callback(s). You can use this mechanism to impose pre-conditions
-on a route, then pass control to subsequent routes if there is no reason to proceed with the current route.
+The callbacks are invoked in the order in which they are provided and behave in the same way as middleware. In order for the next callback to be invoked, the current one must call <code>next()</code>. If a callback doesn't call <code>next()</code> and doesn't respond to the client, the request will hang.
```

Comment:
Hmm, I guess given that it's hard to understand it might be worth rewording to make things even more explicit, for example:

Every callback must either send a response, or call `next()`, allowing one of the subsequent callbacks to send a response. If execution of the series of callbacks ends without sending a response to the client (or erroring), the request will hang.

What do you think?

Reply:
Perfect 👍🏼.

Reply:
Fixed with ba717a8d.